src/pages/en/reference/integrations-reference.md

```diff
@@ -112,6 +114,25 @@ A callback function to add a component framework renderer (i.e. React, Vue, Svel
 - `clientEntrypoint` - path to a file that executes on the client whenever your component is used. This is mainly for rendering or hydrating your component with JS.
 - `serverEntrypoint` - path to a file that executes during server-side requests or static builds whenever your component is used. These should render components to static markup, with hooks for hydration where applicable. [React's `renderToString` callback](https://reactjs.org/docs/react-dom-server.html#rendertostring) is a classic example.
 
+#### `injectRoute` option
+**Type:** `({pattern: string, entryPoint: string}) => void;`
+
+A callback function to inject a route from an integration. Injected routes can be `.astro` pages, or `.js` route handlers.
+
+`injectRoute` takes an object with a `pattern` and an `entryPoint`. 
+
+- `pattern`: The `pattern` is where the route should be output in the browser, for example: `/foo/bar`. `pattern`s reuse Astro's syntax for denoting dynamic routes, for example: `/foo/[bar]`, or `/foo/[...bar]`. Note that a file extension is **not** needed in the `pattern`.
+- `entryPoint`: The `entryPoint` should be a bare module specifier pointing towards the `.astro` page or `.js` route handler that handles the route denoted in the `pattern`.
```

Comment:
Agreeing with @sarah11918 here to avoid repetition and made a few tiny tweaks. (Not sure I _love_ the hyphen separation — would prefer an em-dash or colon — but it’s what the other similar blocks on this page use).

```suggestion
- `pattern` - where the route should be output in the browser, for example `/foo/bar`. A `pattern` can use Astro's filepath syntax for denoting dynamic routes, for example `/foo/[bar]` or `/foo/[...bar]`. Note that a file extension is **not** needed in the `pattern`.
- `entryPoint` - a bare module specifier pointing towards the `.astro` page or `.js` route handler that handles the route denoted in the `pattern`.
```